stable/rendertron/values.yaml

```diff
@@ -0,0 +1,42 @@
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+replicaCount: 1
+image:
+  repository: ravishi/rendertron
+  tag: a6b93ee7ca98687610542ac930597ce5064762d2
```

Comment:
Can you move this to a tag rather than a hash? Possibly a version that correlates to the a software version?